web/src/main/java/org/mskcc/cmo/metadb/web/RequestController.java

```diff
@@ -0,0 +1,66 @@
+package org.mskcc.cmo.metadb.web;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.mskcc.cmo.metadb.service.CmoRequestService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@CrossOrigin(origins = "*")
+@RequestMapping(value = "/")
+@Api(tags = "request-controller", description = "Request Controller")
+public class RequestController {
+    @Autowired
+    private CmoRequestService requestService;
+
+    @Autowired
+    public RequestController(CmoRequestService requestService) {
+        this.requestService = requestService;
+    }
+    
+    /**
+     * fetchRequestGET
+     * @param requestId
+     */
+    @ApiOperation(value = "Retrieve NewCmoRequest",
+        nickname = "fetchRequestGET")
+    @RequestMapping(value = "/request/requestId/{requestId}",
```

Comment:
actually I just realized that this mapping seems a bit weird. 

It should be value = "/request/{requestId}"

and the post should be value = "/request"